import/index_java/src/org/solrmarc/index/VuFindIndexer.java

```diff
@@ -1624,19 +1631,136 @@ protected Double coordinateToDecimal(String coordinateStr) {
      * @return boolean
      */
     protected boolean validateCoordinates(Double west, Double east, Double north, Double south) {
-        if (west == null || east == null || north == null || south == null) {
-            return false;
+        String validLines = "";
+        String validValues = "";
+        String validExtent = "";
+        String validNorthSouth = "";
+        String validCoordDist = "";
+
+        // Validate lines
+        if (validateLines(west, east, north, south)) {
+            validLines = "true";
+        } else {
+            validLines = "false";
+            System.out.println("*** ERROR: Coordinates form a line at the pole...DD coords: "+west+" "+east+" "+north+" "+south);
```

Comment:
Great, thanks! I might have some time to work on this tomorrow, unless you want to make any changes in the meantime. If I do end up changing anything, I'll push my changes here and get your approval before moving forward.